src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
+const ANIMAL_HELTH = 100;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = ANIMAL_HELTH;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide(isHidden = true) {
+    this.hidden = isHidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(target) {
+    const BITE_DMG = 50;
```

Comment:
shortening to consonants is a bad practice, some will understand right away, but others will not
so better rename it to `BITE_DAMAGE`